config.md

```diff
@@ -2,6 +2,7 @@
 
 The container's top-level directory MUST contain a configuration file called `config.json`.
 The canonical schema is defined in this document, but there is a JSON Schema in [`schema/config-schema.json`](schema/config-schema.json) and Go bindings in [`specs-go/config.go`](specs-go/config.go).
+For properties that are only defined for some [protocols](spec.md#protocols), the Go property has a `platform` tag listing those protocols (e.g. `platform:"linux,solaris"`).
```

Comment:
> I don't see why our spec would define such non-platform protocol such as `linux-cgroup` protocol.

I'd floated [making `hooks` a protocol][1] to accommodate a Windows runtime that chose not to support then despite their being possible on Windows (although I'm not sure if hooks are [possible or not on Windows][2]).  And I think we want a `base` protocol to cover stuff that all compliant runtimes will need (e.g. `ociVersion`).  But having a maintainer reply to my counter-argument with a clear rejection of "protocol" is enough for a rebase here.  I'll get it pushed sometime today, and then file a follow-up PR adding `base`, which you can pick up if it looks useful.

[1]: https://github.com/opencontainers/runtime-spec/pull/472#issuecomment-258567371
[2]: https://github.com/opencontainers/runtime-spec/pull/472#issue-157263212